addons/pos_stripe/views/pos_payment_method_views.xml

```diff
@@ -8,6 +8,9 @@
             <xpath expr="//field[@name='use_payment_terminal']" position="after">
                 <!-- Stripe -->
                 <field name="stripe_serial_number" attrs="{'invisible': [('use_payment_terminal', '!=', 'stripe')], 'required': [('use_payment_terminal', '=', 'stripe')]}"/>
+                <div class="mt16" attrs="{'invisible': [('use_payment_terminal', '!=', 'stripe')], 'required': [('use_payment_terminal', '=', 'stripe')]}">
+                    <button name="action_stripe_key" type="object" icon="fa-arrow-right" string="Complete Stripe connect before record a payment method" class="btn-link"/>
```

Comment:
Replace the message by: "Don't forget to complete Stripe connect before using this payment method."